eksconfig/clusterautoscaler.go

```diff
@@ -1,12 +1,18 @@
 package eksconfig
 
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
 // ClusterAutoscalerSpec defines the spec for the Addon
 type ClusterAutoscalerSpec struct {
-	Image          string        `json:"image,omitempty"`
-	CloudProvider  CloudProvider `json:"cloudProvider,omitempty"`
-	MinNodes       int           `json:"minNodes,omitempty"`
-	MaxNodes       int           `json:"maxNodes,omitempty"`
-	ScaleDownDelay string        `json:"scaleDownDelay,omitempty"`
+	Image          string      	 				`json:"image,omitempty"`
+	CloudProvider  CloudProvider 				`json:"cloudProvider,omitempty"`
+	MinNodes       int           				`json:"minNodes,omitempty"`
+	MaxNodes       int           				`json:"maxNodes,omitempty"`
+	Resources      corev1.ResourceRequirements  `json:"resources,omitempty"`
```

Comment:
These json comments should be aligned. Do you have a go formatter installed?

Reply:
@njtran You can run `./hack/fmt.sh`?